fhir/scripts/populate.js

```diff
@@ -0,0 +1,149 @@
+const asyncHandler = require('../src/lib/async-handler');
+const { COLLECTION } = require('../src/constants');
+const { mongoConfig } = require('../src/config');
+const package = require('../package.json');
+const mongo = require('../src/lib/mongo');
+const program = require('commander');
+const path = require('path');
+const glob = require('glob');
+
+// const {
+// 	PatientSchema,
+// 	ObservationSchema
+// } = require('fhir-schemas');
+
+// const SCHEMA_MAP = {
+// 	'Patient': PatientSchema,
+// 	'Observation': ObservationSchema
+// };
+
+const VALID_PROFILE_NAMES = [
+	COLLECTION.OBSERVATION,
+	COLLECTION.PATIENT
+];
+
+/**
+ * @function loadProfiles
+ * @summary Load the profiles into Mongo and log status as we go
+ */
+let loadProfiles = async args => {
+	let { profiles, reset } = args;
+	// Connect to Mongo
+	let [ err, client ] = await asyncHandler(mongo(mongoConfig.connection, mongoConfig.options));
+
+	// Fail if ther is an error
+	if (err) {
+		console.error(err);
+		process.exit(err);
```

Comment:
Shouldn't this be an int rather than a string?

Reply:
Yea, actually meant to throw the error, forgot to update this